modules/rgbd/src/tsdf.cpp

```diff
@@ -462,30 +383,6 @@ struct IntegrateInvoker : ParallelLoopBody
     Mat pixNorms;
 };
 
-static cv::Mat preCalculationPixNorm(Depth depth, const Intr& intrinsics)
-{
-    int height = depth.rows;
-    int widht = depth.cols;
-    Point2f fl(intrinsics.fx, intrinsics.fy);
-    Point2f pp(intrinsics.cx, intrinsics.cy);
-    Mat pixNorm (height, widht, CV_32F);
-    std::vector<float> x(widht);
-    std::vector<float> y(height);
-    for (int i = 0; i < widht; i++)
-        x[i] = (i - pp.x) / fl.x;
-    for (int i = 0; i < height; i++)
-        y[i] = (i - pp.y) / fl.y;
-
-    for (int i = 0; i < height; i++)
-    {
-        for (int j = 0; j < widht; j++)
-        {
-            pixNorm.at<float>(i, j) = sqrtf(x[j] * x[j] + y[i] * y[i] + 1.0f);
-        }
-    }
-    return pixNorm;
-}
-
 // use depth instead of distance (optimization)
 void TSDFVolumeCPU::integrate(InputArray _depth, float depthFactor, const cv::Matx44f& cameraPose,
                               const Intr& intrinsics)
```

Comment:
Since `integrateVolumeUnit` is in tsdf_functions, the TSDF integration should be rewritten using it. SIMD version is to be taken too.
If this amount of work is too big for that PR, then it's better to postpone it till next PR.